packages/database/data-migrations/20240408123456_userid_migration/data-migration.ts

```diff
@@ -0,0 +1,64 @@
+import { PrismaClient } from "@prisma/client";
+
+const prisma = new PrismaClient();
+
+async function main() {
+  await prisma.$transaction(async (tx) => {
+    // get all the persons that have an attribute class with the name "userId"
+    const personsWithUserIdAttribute = await tx.person.findMany({
+      where: {
+        attributes: {
+          some: {
+            attributeClass: {
+              name: "userId",
+            },
+          },
+        },
+      },
+      include: {
+        attributes: {
+          include: { attributeClass: true },
+        },
+      },
+    });
+
+    for (let person of personsWithUserIdAttribute) {
+      // If the person already has a userId, skip it
+      if (person.userId) {
+        continue;
+      }
+
+      const userIdAttributeValue = person.attributes.find((attribute) => {
+        if (attribute.attributeClass.name === "userId") {
+          return attribute;
+        }
+      });
+
+      if (!userIdAttributeValue) {
+        continue;
+      }
+
+      await tx.person.update({
+        where: {
+          id: person.id,
+        },
+        data: {
+          userId: userIdAttributeValue.value,
+        },
+      });
+    }
+
+    // Delete all attributeClasses with the name "userId"
+    await tx.attributeClass.deleteMany({
+      where: {
+        name: "userId",
+      },
+    });
+  });
```

Comment:
we need to add a timeout, at least:
```
{
  timeout: 60000,
}
```
maybe even more, else it will fail on large databases with many people (like formbricks cloud).